src/main/java/com/takipi/oss/storage/caching/StringUtil.java

```diff
@@ -0,0 +1,11 @@
+package com.takipi.oss.storage.caching;
```

Comment:
Move from caching package to some util package

Reply:
done